alerts/AggregatedAPIErrors.md

```diff
@@ -0,0 +1,65 @@
+# AggregatedAPIErrors
+
+## Meaning
+
+This alert is triggered in case multiple calls on the aggregated API of OpenShift failed over a certain time window.
+
+## Impact
+Errors on the aggregated API can result in the unavailability of some OpenShift services.  
+
+## Diagnosis
+
+##### Check the APIs status checks are on True
+
+Currently, there are three aggregated APIs in an OpenShift Cluster. The API on the `openshift-apiserver` namespace, the prometheus-adapter on the namespace `openshift-monitoring`, and the package-server service in the `openshift-operator-lifecycle-manager` namespace.
+
+To get a list of the aggregated APIs and the `APIGroup` to whom they answer requests use the following command:
+
+```shell
+$ oc get apiservice -n openshift-apiserver | grep openshift-
```

Comment:
`apiservice` is not a namespaced resource as such so `-n openshift-apiserver` can be omitted.